app/controllers/sessions_controller.rb

```diff
@@ -1,14 +1,17 @@
 class SessionsController < Devise::SessionsController
-    before_action :authenticate_user!
     respond_to :json
-  
-    private
-  
+    before_action :rewrite_param_names, only: [:create]
+    def create
+      super
+    end
     def respond_with(resource, _opts = {})
       render json: resource
     end
   
     def respond_to_on_destroy
       head :no_content
     end
+    def rewrite_param_names
+        request.params[:user] = {email: request.params[:session][:email],password: request.params[:session][:password]}
```

Comment:
```suggestion
        request.params[:user] = { email: request.params[:session][:email], password: request.params[:session][:password] }
```
The logic is perfect! But rubocop should have warned you about those missing spaces, and that the line is longer than 100 characters.